client/my-sites/domains/domain-management/index.jsx

```diff
@@ -23,19 +19,15 @@ export default {
 	AddDnsRecord,
 	ContactsPrivacy,
 	DnsRecords,
-	DnsRecordsList,
```

Comment:
The component is still used, but this particular reference wasn't being used anywhere.